### PR TITLE
update blobfuse driver to v1.4.3

### PR DESCRIPTION
**Pull request by andyzhangx**
blobfuse v1.4.3 is a stable version released 1/12/2022

### CODE REVIEW

**Comment by andyzhangx** [comment]
set up test cluster failed:
```
2022-03-07T02:34:31.4914794Z + az aks get-credentials -g agentbaker-e2e-tests -n agentbaker-e2e-test-cluster --file kubeconfig --overwrite-existing
2022-03-07T02:34:32.6161325Z ERROR: (ControlPlaneNotFound) List credential operation conflicts with failed cluster, please resolve failed cluster issue and see https://aka.ms/aks-cluster-failed for resolution.
2022-03-07T02:34:32.6162572Z Code: ControlPlaneNotFound
2022-03-07T02:34:32.6163811Z Message: List credential operation conflicts with failed cluster, please resolve failed cluster issue and see https://aka.ms/aks-cluster-failed for resolution.
```

**Comment by andyzhangx** [comment]
> set up test cluster failed: https://msazure.visualstudio.com/09706533-03bf-4b43-9a9b-b49c75429646/_apis/build/builds/52699265/logs/47
> 
> ```
> 2022-03-07T02:34:31.4914794Z + az aks get-credentials -g agentbaker-e2e-tests -n agentbaker-e2e-test-cluster --file kubeconfig --overwrite-existing
> 2022-03-07T02:34:32.6161325Z ERROR: (ControlPlaneNotFound) List credential operation conflicts with failed cluster, please resolve failed cluster issue and see https://aka.ms/aks-cluster-failed for resolution.
> 2022-03-07T02:34:32.6162572Z Code: ControlPlaneNotFound
> 2022-03-07T02:34:32.6163811Z Message: List credential operation conflicts with failed cluster, please resolve failed cluster issue and see https://aka.ms/aks-cluster-failed for resolution.
> ```

@mainred `Agentbaker E2E` is broken? do you know who can fix this?

**Comment by andyzhangx** [comment]
@mainred could you approve? thanks.

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ installDeps() {
     aptmarkWALinuxAgent hold
     apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
     apt_get_dist_upgrade || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
-    BLOBFUSE_VERSION="1.4.2"
+    BLOBFUSE_VERSION="1.4.3"
```
Comment (mainred) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:27]: do we need to cache `1.4.2` as well, before it's not used in AKS?

**Review by mainred — APPROVED** [review]
left a nit

**Comment by mainred** [comment]
sorry for the late response, the test failed for a failed cluster, I think it should have recovered after deleting the cluster after a offline sync with Zhecheng.

**Pull request merged**